Package.swift

```diff
@@ -32,6 +32,7 @@ let package = Package(
             exclude: ["Support"],
             cSettings: [
                 .define("APP_CENTER_C_NAME", to: "\"appcenter.ios\"", .when(platforms: [.iOS])),
+                .define("APP_CENTER_C_NAME_CATALYST", to: "\"appcenter.ios\"", .when(platforms: [.macOS])),
```

Comment:
Let's hold this one. See AB#80703

Reply:
btw do we really need this custom define? I guess it will be easier just define it in header with preprocessor instead of xcfiles and SPM. I do not really like introducing new custom define name

Reply:
I think in any case we probably don't need a new define.